change/@ni-nimble-components-21de2fc8-dfc6-4249-9098-d7842ce0eff3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Refactored rich text components to use common models",
+  "packageName": "@ni/nimble-components",
+  "email": "sai.krishnan.perumal@ni.com",
```

Comment:
nit: If you prefer not to expose your email address to the world, [configure GitHub to "Keep my email address private"](https://github.com/settings/emails) before generating the change file.